_po/ja/how-to/laravel.po

```diff
@@ -337,59 +496,81 @@ msgid ""
 "    {\n"
 "        $posts = \\App\\Models\\Post::query()->paginate(10);"
 msgstr ""
+"class PostController extends Controller\n"
+"{\n"
+"    public function index() : View\n"
+"    {\n"
+"        $posts = \\App\\Models\\Post::query()->paginate(10);"
 
 msgid ""
 "        return view('posts.index', compact('posts'));\n"
 "    }"
 msgstr ""
+"        return view('posts.index', compact('posts'));\n"
+"    }"
 
 msgid ""
 "    public function search(Request $request) : View\n"
 "    {\n"
 "        $keyword = $request->input('keyword');"
 msgstr ""
+"    public function search(Request $request) : View\n"
+"    {\n"
+"        $keyword = $request->input('keyword');"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.